full-stack-practice/client/src/pages/Catalog.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { ProductCard } from '../components/ProductCard';
+
+export type Product = {
+  productId: number;
+  name: string;
+  price: number;
+  imageUrl: string;
+  shortDescription: string;
+  longDescription: string;
+};
+
+export function Catalog() {
+  const [products, setProducts] = useState<Product[]>([]);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Don't add eslint rules without asking me. Chances are if you are getting a lint error it's because of a mistake you're making. In this case, that means you aren't using a variable that you should be using